contracts/compliance/modular/modules/MinTransferByCountryModule.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.27;
+
+import "../IModularCompliance.sol";
+import "../../../token/IToken.sol";
+import "./AbstractModuleUpgradeable.sol";
+
+ event MinimumTransferAmountSet(uint16 indexed country, uint256 amount);
+
+
+/**
+ * @title MinTransferByCountry Module
+ * @dev Enforces minimum transfer amounts for token holders from specified countries
+ * when creating new investors for that country
+ */
+contract MinTransferByCountryModule is AbstractModuleUpgradeable {
+
+    mapping(address compliance => mapping(uint16 country => uint256 minAmount)) private _minimumTransferAmounts;
+    
+    function initialize() external initializer {
+        __AbstractModule_init();
+    }
+
+    /**
+     * @dev Sets minimum transfer amount for a country
+     * @param country Country code
+     * @param amount Minimum transfer amount
+     */
+    function setMinimumTransferAmount(uint16 country, uint256 amount) external onlyComplianceCall {
+        _minimumTransferAmounts[msg.sender][country] = amount;
+
+        emit MinimumTransferAmountSet(country, amount);
```

Comment:
the event has to specify msg.sender too (the compliance for which it has been set)